extensions/wikia/AdEngine/js/lookup/prebid/adapters/fastlane.js

```diff
@@ -0,0 +1,162 @@
+/*global define*/
+define('ext.wikia.adEngine.lookup.prebid.adapters.fastlane', [
```

Comment:
Reporting. We decided to go with separate bidder when we implemented Rubicon Video